ansible/roles/cli_minio/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+
+- name: Create minio directory
+  ansible.builtin.file:
+    path: /opt/minio/
+    state: directory
+    owner: deerhide-operator
+    group: deerhide-operator
+    mode: '0755'
+  become: true
+
+- name: Found and register "arch"
+  ansible.builtin.command:
+    cmd: dpkg --print-architecture
+  become: true
+  register: archi
```

Comment:
'arch' please, as suggested in the task name.

Reply:
It's fixed